app/src/components/mode-specific/redirects/NavigateToAppModesRootRoute.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
Can we rename this to `RootComponent` and move this out of `mode-specific`. This can handle our home page in future.